setup.py

```diff
@@ -65,7 +65,7 @@
     "fastapi>=0.68.0",
     "uvicorn[standard]>=0.14.0,<1",
     "gunicorn; platform_system != 'Windows'",
-    "dask[dataframe]>=2024.4.2",
+    "dask[dataframe]>=2024.2.1",
```

Comment:
are we downgrading to support pandas 1?

Reply:
not downgrading exactly, just lower the bound of dask to relax the pandas version. I think force pandas to be 2.0+ maybe too tight, will cause conflicts with many other projects.